transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessengerTransportImpl.kt

```diff
@@ -1,32 +1,19 @@
 package com.genesys.cloud.messenger.transport.core
 
-import com.genesys.cloud.messenger.transport.network.DEFAULT_PING_INTERVAL_IN_SECONDS
 import com.genesys.cloud.messenger.transport.network.DeploymentConfigUseCase
 import com.genesys.cloud.messenger.transport.network.PlatformSocket
 import com.genesys.cloud.messenger.transport.network.ReconnectionHandlerImpl
 import com.genesys.cloud.messenger.transport.network.WebMessagingApi
 import com.genesys.cloud.messenger.transport.shyrka.receive.DeploymentConfig
-import com.genesys.cloud.messenger.transport.util.DefaultTokenStore
 import com.genesys.cloud.messenger.transport.util.logs.Log
 import com.genesys.cloud.messenger.transport.util.logs.LogTag
 
-/**
- * Client entry point for the Mobile Messenger Transport SDK.
- */
-object MobileMessenger {
+internal class MessengerTransportImpl(private val configuration: Configuration) {
```

Comment:
Wondering about the "Impl" naming here as this doesn't seem to implement any interfaces.

Reply:
Will use `...Internal` as recommended by Anton. In the past it _did_ implement an interface but that was a few changes ago.